PyRDF/backend/Local.py

```diff
@@ -16,9 +16,7 @@ def local_executor(generator):
 
     output = mapper(TDF)
 
-    node_map = generator.action_node_map
+    output[0][0].GetValue() # Trigger event-loop
```

Comment:
It would be more pythonic and clearer to decompose the tuple in line 17: `values, nodes = mapper(TDF)` and here you can just do `values[0].GetValue()`.

Reply:
And you can also take the opportunity to rename variable `TDF` to `tdf` :smile:

Reply:
@etejedor , I guess `rdf` would be a great fit